### DISCUSSION TITLE
Some problems after FW update

### DISCUSSION THREAD

**Discussion by Micha213** [darthcloud/BlueRetro#261] [Q&A]
I updated my PS1 receiver to the latest FW a few days ago. The previous FW was quite old, I think it was v0.19.1. I used OTA for the update.

After the update, however, I was still unable to connect my new Xbox Series X and my Xbox Elite 2 controller (both current FW). I did a reset (boot button pressed for 10 seconds). After that everything was a mess. The new Xbox controllers still haven't connected. Older Xbox and PS4 controllers can be connected, the button assignment was messed up.

I then flashed the FW again via USB. But this time all three dates. This solved the problem with the newer Xbox controllers. But the settings were still messed up.

Has the way the settings are saved changed at some point between the FW versions? If so, is there a way I can do a clean installation or reset the settings?

**Comment by darthcloud** [comment]
Factory reset will revert FW to original one flashed at 0x10000 (0.19.1 in your case).

Their is upgrade code to update the config or to reset it on downgrade. I do test this with PSUnoRetro and haven't got corruption myself.

Anyway the easiest way to reset it is to write an empty partition at 0x310000 with the ESP flash_download_tool
[empty_spiffs.zip](https://github.com/darthcloud/BlueRetro/files/8520495/empty_spiffs.zip)

Soon I will make a file explorer over BLE and that will become the way to reset the config.
